src/freenet/crypt/JceLoader.java

```diff
@@ -55,8 +68,20 @@ static private boolean checkUse(String prop, String def)
 		}
 		BouncyCastle = p;
 		// optional
+		if (checkUse("use.SunJCE")) {
+		    try{
+		        KeyGenerator kgen = KeyGenerator.getInstance("AES", "SunJCE");
+		        kgen.init(256);
+		    }
+		    catch(Throwable e) {
+		        final String msg = "Error with SunJCE. Unlimited policy file not installed.";
```

Comment:
That makes sense. I think at the time I was just copying convention used at the file, but I'll change it.